### DISCUSSION TITLE
Link UP/DOWN not smooth and when to select MTCP Master/Enabled/Backup

### DISCUSSION THREAD

**Discussion by pbix** [Ysurac/openmptcprouter#2249] [Q&A]
I have had good success getting OpenMTCPRouter up and running.  Thanks to Ysurac for a nice interface to help us newbies enjoy some success.  When both links are up and stable it seems to work Ok.

I am bonding a WiMax link with a Starlink.  These links can be unstable at times and when one goes down there does not seem to be a smooth transition to excluding the down link.  I have been forced to disable the one that is not working well at the moment to allow things like live meetings proceed smoothly.  This is what I was hoping MTCP would excel at so I am confused.

Also, I have read the documentation and the discussions but have not yet seen anything regarding selecting Master vs. Enabled vs. Backup for the Multipath TCP setting.  Particularly when to select Enabled vs Backup.  Also in what ways is the "Master" link different than the other links.  Why is a master needed?

Trying to understand what I can do to improve the switch over and recovery of the down link.  Would really appreciate some suggestions on how to configure for maximum reliability.

I am using Raspberry Pi4 and already checked that CPU usage on that and my VPS are below 25%.  Also my total available bandwidth is around 250mbps in the best of circumstances and my VPS support 1 tbps so not thinking that is a limitation.

**Comment by Chris5342** [comment]
https://github.com/Ysurac/openmptcprouter/discussions/2357#discussioncomment-2833963 Might help.

I have started to try SQM to solve this type of issue.
